patches/0007-Add-backend-code-gen.patch

```diff
@@ -378,10 +378,10 @@ index ad6081552af15d..d5948dbc5f8a2a 100644
  // Do not edit the build constraint by hand. It is generated by "backendgen.go".
  
 -//go:build ignore
-+//go:build !(goexperiment.boringcrypto && linux && cgo && (amd64 || arm64) && !android && !msan) && !(goexperiment.cngcrypto && windows) && !(goexperiment.opensslcrypto && linux && cgo)
- 
++//go:build !(goexperiment.boringcrypto && (linux || darwin) && cgo && (amd64 || arm64) && !android && !msan) && !(goexperiment.cngcrypto && windows) && !(goexperiment.opensslcrypto && (linux || darwin) && cgo)
```

Comment:
I don't get the same result when I run `go/bin/go generate crypto/internal/backend`. I think you need to fix the earlier patch, but make sure you're using the generator. (See the top of the patch for more info on what this is all about.)